docs/release-process.md

```diff
@@ -0,0 +1,21 @@
+**Releasing a new version**
+
+Semanting versioning is used to release different versions of the action. Following steps are to be followed :
+
+1. Create a new branch for every major version. \
```

Comment:
For major versions, this branch should be created from main branch (and cherry pick changes from master only when there are additional changes on master happened after release branch is forked). Ensure node_modules are added.